src/views/DetailedView/useVenueDetails.tsx

```diff
@@ -0,0 +1,14 @@
+import { useEffect, useState } from "react";
+import { VenueDetails } from "../../shared/types/VenueDetails.ts";
+import { axiosInstance } from "../../shared/utils/axiosInstance.ts";
+
+export const useVenueDetails = (venueId: number) => {
+  const [venueDetails, setVenueDetails] = useState<VenueDetails | null>(null);
+  useEffect(() => {
+    axiosInstance
+      .get<VenueDetails[]>(`/venuesDetails?venueId=${venueId}`)
```

Comment:
Create file `shared/api/venuesDetailsApi.ts` with all the venuesDetails-related api calls. You can look into my example to get the inspiration.